src/thread.h

```diff
@@ -56,6 +56,7 @@ class Thread {
   void idle_loop();
   void start_searching();
   void wait_for_search_finished();
+  int bestMoveCount(Move move);
```

Comment:
I think the usual SF style for function names is to use "_"  Something like best_move_count().

Reply:
Yes, you are right. For my own coding style i prefer upper-lower cases instead of underscores.